### ISSUE TITLE
Installer or Chocolatey package 

### ISSUE THREAD

**Issue by tiksn** [mbdavid/LiteDB.Studio#3]
It would be nice to have installer of Chocolatey package.

**Comment by mbdavid** [comment]
Hi @tiksn, I will read more about Chocolatey package to discover how publish there. Thanks!

**Comment by digitalcoyote** [comment]
I'd be willing to create/maintain a package for this project. I'll take a look at this when I get a free moment (got a couple irons in the fire).

**Comment by digitalcoyote** [comment]
I've got a package submitted for this project (https://chocolatey.org/packages/litedb.studio). It may take a bit to be approved. If you would like, I can add you as a maintainer (requires a Chocolatey account) to give you management rights (delist problematic versions, manually push additional versions, etc). Once this get's approved, I'll try to get a PR ready with installation instructions for the readme.

**Comment by digitalcoyote** [comment]
Do you want an msi as well? I can  contribute the code for one using [wix#](https://www.codeproject.com/Articles/1231849/Creating-a-Wixsharp-Installer-that-Includes-Prereq).

**Comment by mbdavid** [comment]
Hi @digitalcoyote, thanks for this. I will release new version tomorrow (0.6) and version 1.0 in Jan/31.

**Comment by digitalcoyote** [comment]
[LiteDB-Studio](https://chocolatey.org/packages/litedb-studio has been approved.

**Comment by igitur** [comment]
While this issue is still open, can I add a request to add it as a scoop package? ( https://scoop.sh )

**Comment by rbeesley** [comment]
@igitur, agreed. Two or three years ago, all I wanted was Chocolatey. However for a lot of my dev tools, I find that Scoop serves me better. There are still some things for which I'll go back, but if you aren't installing globally, scoop just seems to be so much less invasive.
